hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/contract/ITestOzoneContractSeek.java

```diff
@@ -19,21 +19,34 @@
 package org.apache.hadoop.fs.ozone.contract;
 
 import java.io.IOException;
+import java.util.Collection;
 
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.contract.AbstractContractSeekTest;
 import org.apache.hadoop.fs.contract.AbstractFSContract;
 
 import org.junit.AfterClass;
-import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 
 /**
  * Ozone contract tests covering file seek.
  */
+@RunWith(Parameterized.class)
 public class ITestOzoneContractSeek extends AbstractContractSeekTest {
-  @BeforeClass
-  public static void createCluster() throws IOException {
-    OzoneContract.createCluster();
+
+  private static boolean fsOptimisedClient;
+  private static boolean fsOptimisedServer;
+
+  public ITestOzoneContractSeek(boolean fsOptimClient,
+      boolean fsOptimServer) throws IOException {
+    if (fsOptimisedClient != fsOptimClient ||
+        fsOptimisedServer != fsOptimServer) {
```

Comment:
Typo: `fsOptimClient` -> we can simplify it like `fsoClient` and `fsoServer`. Please update all the occurrences. Thanks!

Reply:
Done.